src/main/java/org/owasp/validator/html/AntiSamy.java

```diff
@@ -145,8 +148,8 @@ public CleanResults scan(Reader reader, Writer writer, Policy policy) throws Sca
   }
 
   /**
-   * This method wraps <code>scan()</code> using the Policy in the specified file. Uses the DOM
-   * parser for scanning.
+   * This method scans the supplied HTML input and produces clean/sanitized results per the supplied
+   * AntiSamy policy file using the DOM parser.
    *
    * @param taintedHTML Untrusted HTML which may contain malicious code.
    * @param filename The file name of the custom policy to enforce.
```

Comment:
@davewichers - Since you changed the formal parameter name to 'policyFilename', you should probably change it here as well.